dockstore-cli-integration-testing/pom.xml

```diff
@@ -268,6 +268,8 @@
                         --add-opens java.base/java.base=ALL-UNNAMED
                         --add-opens java.base/java.lang.reflect=ALL-UNNAMED
                         --add-opens java.base/sun.nio.fs=ALL-UNNAMED
+                        <!-- for  Java 21, see   -->
```

Comment:
Is this comment incomplete? See what?

Reply:
extended comment, needed for system stubs in Java 21